src/droplet.F

```diff
@@ -1076,10 +1071,21 @@ subroutine BE_integration(dt,dbldt,np,pdata,pdata_locind,xf,yf, &
             pdata(np,pract) = -1.0  !Signal this as no longer alive
             num_fallout = num_fallout + 1
             !Collect some statistics on those falling out:
+
             !$acc atomic update
             num_destroyed = num_destroyed + pdata(np,prmult)
+
             !$acc atomic update
```

Comment:
I don't understand why we are using an atomic update here.  Isn't this an acc reduction operation?

Reply:
Yes, this is a reduction operation. My concern is that this operation is done inside `BE_integration` routine, while the loop of `np` is done inside the `droplet_driver` routine, which calls the `BE_integration` routine per loop iteration. I am not sure if the acc reduction clause works properly in this case.

Reply:
@johnmauff I have run a new GPU test on Derecho with the `acc reduction` clause rather than the `acc atomic` directive. The simulation finishes successfully and the verification result remains the same. However, the affected variables (e.g., `num_destroyed`) are not included in the droplet diagnostic. Maybe @drichte2 can help look at the new code and see if the results are ok?